res/css/categories.css

```diff
@@ -6,17 +6,20 @@
   --category-color-transparent: #fff;
   --category-color-purple: var(--purple-70);
   --category-color-green: var(--green-60);
-  --category-color-orange: var(--orange-60);
+  --category-color-orange: var(--orange-50);
   --category-color-lightblue: var(--blue-40);
   --category-color-blue: var(--blue-60);
-  --category-color-brown: var(--yellow-70);
+  --category-color-brown: var(--orange-70);
+  --category-color-magenta: var(--magenta-60);
 
   /* it is hard to create a lighter version of the following colors  */
```

Comment:
this distinction was also implemented as part of #4193, for an initial version of the patch, but in the end it wasn't used. What do you think about removing the comment and merging them in the same list?

Reply:
Yeah, it sounds good to me.